internal/config/progress.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"os"
+
+	"github.com/UpCloudLtd/progress"
+	"github.com/UpCloudLtd/progress/messages"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+func GetProgressOutputConfig() *progress.OutputConfig {
```

Comment:
Could put these settings in `messages.GetDefaultOutputConfig()`  in `progress` repo and call directly that? Or get the current default and only override the changes here.

Reply:
Done. This now uses default config as base and colors whole message with bright colors.